x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -609,7 +619,59 @@ export const RulesList: React.FunctionComponent = () => {
         sortable: false,
         truncateText: false,
         'data-test-subj': 'rulesTableCell-interval',
-        render: (interval: string) => formatDuration(interval),
+        render: (interval: string, item: RuleTableItem) => {
+          const durationString = formatDuration(interval);
+          return (
+            <>
+              <EuiFlexGroup direction="row" gutterSize="xs">
+                <EuiFlexItem grow={true}>{durationString}</EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  {item.showIntervalWarning && (
+                    <EuiToolTip
+                      data-test-subj={`ruleInterval-config-tooltip-${item.index}`}
+                      title={i18n.translate(
+                        'xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.intervalTooltipTitle',
+                        {
+                          defaultMessage: 'Below configured minimum interval',
+                        }
+                      )}
+                      content={i18n.translate(
+                        'xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.intervalTooltipText',
+                        {
+                          defaultMessage:
+                            'Rule interval of {interval} is below the minimum configured interval of {minimumInterval}. This may impact alerting performance.',
+                          values: {
+                            minimumInterval: formatDuration(
+                              config.minimumScheduleInterval!.value,
+                              true
+                            ),
+                            interval: formatDuration(interval, true),
+                          },
+                        }
+                      )}
+                      position="top"
+                    >
+                      <EuiButtonIcon
+                        color="text"
+                        data-test-subj={`ruleInterval-config-icon-${item.index}`}
+                        onClick={() => {
+                          if (item.isEditable && isRuleTypeEditableInContext(item.ruleTypeId)) {
+                            onRuleEdit(item);
+                          }
+                        }}
+                        iconType="iInCircle"
```

Comment:
Let's change this to `flag`.  After seeing this more in action the `iInCircle` doesn't feel correct.